build/yaml/build-code-coverage-w-coverlet.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+  - master
+
+pr:
+  - master
+
+pool:
+  vmImage: 'windows-2019'
+
+variables:
+  BuildPlatform: 'Any CPU'
+  BuildConfiguration: 'debug'
+  ProjectName: 'Microsoft.Bot.Builder.Adapters.Twilio.Tests'
+  Build.SourcesDirectory: ''
+  OutputFile: 'result.xml'
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 2.1.x'
+  inputs:
+    version: 2.1.x
+
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 3.0.x'
+  inputs:
+    version: 3.0.x
+
+- powershell: |
+   dotnet tool install --global coverlet.console
+  displayName: 'Install Coverlet'
+
+- task: NuGetToolInstaller@0
+  displayName: 'Use NuGet 4.9.1'
+  inputs:
+    versionSpec: 4.9.1
+
+- task: NuGetCommand@2
+  displayName: 'NuGet restore'
+  inputs:
+    restoreSolution: '$(System.DefaultWorkingDirectory)\Microsoft.Bot.Builder.sln'
+
+- task: VSBuild@1
+  displayName: 'Build solution Microsoft.Bot.Builder.sln'
+  inputs:
+    solution: Microsoft.Bot.Builder.sln
+    vsVersion: 16.0
+    platform: '$(BuildPlatform)'
+    configuration: '$(BuildConfiguration)'
+
+- powershell: |
+   coverlet Adapters\$(ProjectName)\bin\$(BuildConfiguration)\netcoreapp3.0\$(ProjectName).dll --target "dotnet" --targetargs "test Adapters\$(ProjectName)\$(ProjectName).csproj -v n --configuration debug --no-build --no-restore --filter TestCategory!=IgnoreInAutomatedBuild" --format cobertura --output $(Build.SourcesDirectory)\$(OutputFile) --exclude-by-file "Adapters\$env:ProjectName\TwilioClientWrapper.cs" --exclude-by-file "Adapters\$env:ProjectName\TwilioMessage.cs" 
```

Comment:
Consider using the filter '--filter "TestCategory!=IgnoreInAutomatedBuild&TestCategory!=FunctionalTests"'.